docker/db-migration/migrations/014-remove-flow-group-id-field.yaml

```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: tag
+      author: dpoltavets
+      changes:
+        - tagDatabase:
+            tag: 014-remove-flow-group-id-field
+
+  - changeSet:
+      id: remove_flow_group_id_field
+      author: dpoltavets
+      changes:
+        - sql: "DROP INDEX flow.group_id"
```

Comment:
We must delete fileds itself (i.e. `UPDATE flow REMOVE group_id`).

Reply:
This command has been added